CONTRIBUTING.md

```diff
@@ -2,8 +2,6 @@
 
 New contributors are always welcome, when it doubt please ask questions. We strive to be an open and welcoming community. Please be nice to one another.
 
-Please read the [code of conduct](CODE_OF_CONDUCT.md)
```

Comment:
Could you point to the foreman community guidelines instead here and in the other relevant places?

Reply:
Done. I cleaned out the README a bit, because i think the reason to have a `CONTRIBUTING.md` is to just link to it in a `README`, I hope that's OK.

Reply:
No problem to remove this code of conduct. I will apply foreman team rules of course.